src/org/ods/orchestration/usecase/BitbucketTraceabilityUseCase.groovy

```diff
@@ -115,10 +115,9 @@ class BitbucketTraceabilityUseCase {
                 nextPageStart = pullRequests.nextPageStart
             }
 
-            records += pullRequests.values.collect { pullRequest ->
-                processPullRequest(token, repo, pullRequest)}
-                .flatten()
-
+            records += pullRequests.values.collectMany { pullRequest ->
```

Comment:
This code was updated in: https://github.com/opendevstack/ods-jenkins-shared-library/pull/1085 but that change isn't in 4.x branch yet. So we have to get in mind in case of release this before the other.

Reply:
Thanks for the reminder; we will take it into account.